app/templates/formbuilder/edit.hbs

```diff
@@ -11,10 +11,18 @@
 
 <main id="main" class="au-c-main-container">
   <div class="au-c-main-container__content">
+    <div class="playground__tab">
+      Builder
```

Comment:
The toggle is a bit weird since you can only click the "validation" label to toggle it. Not sure what the fix for that would be though so something for the UX team in one of the future sprints.

Reply:
Indeed putting in time and effort now to make it work is a bit of a waste. On Monday I also sat down with Marie and there we said that we are going to take a look on how we are going to organise everything within the form builder design wise & placements.